spacy/kb.pyx

```diff
@@ -343,7 +372,7 @@ cdef class Writer:
             loc = bytes(loc)
         cdef bytes bytes_loc = loc.encode('utf8') if type(loc) == unicode else loc
         self._fp = fopen(<char*>bytes_loc, 'wb')
-        assert self._fp != NULL
+        assert self._fp != NULL, "Could not access %s" % loc
```

Comment:
I'm a repeat offender at this, but we should try to avoid `assert` for expected errors. Unable to access a file is definitely something that can come up, so I think this should be a conditional and raise either `IOError` or `OSError`.